src/aks-preview/azext_aks_preview/agentpool_decorator.py

```diff
@@ -618,6 +618,17 @@ def get_gateway_prefix_size(self) -> Union[int, None]:
         """
         return self.raw_param.get('gateway_prefix_size')
 
+    def get_vm_sizes(self) -> List[str]:
+        """Obtain the value of vm_sizes.
+        :return: list of strings
+        """
+        raw_value = self.raw_param.get("vm_sizes")
+        if raw_value is not None:
+            vm_sizes = raw_value.split(",")
+        else:
+            vm_sizes = [self.get_node_vm_size()]
```

Comment:
What is the expected behavior if user specifies vm_sizes and node_vm_size at the same time? Would sever refuse request or take some preference on these two properties?

BTW, these 2 options (`--vm-sizes`, `--node-vm-size`) are doing similar things but named differently? Better add the `node` prefix for the newly added option, could confirm with PM.

Reply:
as shown in codes, we use value from `--vm-sizes` first, if it's not specified, we use the value from `--node-vm-size` (or the default value of it).

Reply:
the 2 options are different. `--node-vm-size` only accepts one value. VirtualMachines agentpool can have one or more than one vmsize so we add new parameter `--vm-sizes` for it. We just let the old parameter also work for our scenario.

Reply:
If you can do the necessary changes that will be appreciated.

Reply:
If there's no strong reason we prefer not to add node- prefix, mostly because
1.it matches the field name in api and no confusion on what vm size is for.
2.less likely to be confused with node-vm-size which only takes 1 size.

Reply:
Pls bug fixes